robot/rospackages/src/mux_selector/scripts/mux_select_server.py

```diff
@@ -115,7 +115,7 @@ def select_device(device):
 def mux_select_server():
     global local
     local = False
-    if len(sys.argv) == 2:
+    if len(sys.argv) > 3:
```

Comment:
This scheme works well with the launch file. Although when you want to run it in standalone, if you try `rosrun mux_selector mux_selector_server.py local` then it no longer works. What I recommend is scraping this scheme of checking the amount of arguments and instead iterating over each argument to check if it's an exact match of "local"